erpnext_com/www/docs/user/manual/en/selling/sales-order.md

```diff
@@ -15,13 +15,17 @@ There are various transactions which can be triggered from Sales Order like;
 <img class="screenshot" alt="Sales Order flow" src="{{docs_base_url}}/assets/img/selling/sales-order-f.jpg">
 
 ### 1. How to create a Sales Order
-1. Go to **Selling > Sales > Sales Order > New Sales Order**.
+1. Go to **Selling > Sales > Sales Order > New Sales Order** or Type 'New Sales Order' into the Global Search field on the Top of the page, left of the username/icon
```

Comment:
line 18: end the sentence at 'global search field'. We're trying to keep the steps concise for a quick start but I see you've added good info in other steps.

Reply:
> _line 18: end the sentence at 'global search field'. We're trying to keep the steps concise for a quick start_
I personally prefer clarity even if it is  less condensed. Anyway, I'll adjust and resend as soon I have figured out how to do that (cancel PR, send a new one? or can I make a new commit on my branch that then goes into the existing PR?)